.jenkins

```diff
@@ -111,7 +111,7 @@ pipeline {
                                 -DCMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -Wno-deprecated-declarations -Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-unknown-cuda-version -Wno-sycl-target" \
                                 -DKOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED=0 \
                                 -DKokkos_ARCH_NATIVE=ON \
-                                -DKokkos_ARCH_VOLTA70=ON \
```

Comment:
I just got too annoyed with surprising restrictions on `Volta70`, especially given that all the machines with an Nvidia GPU I have access to for debugging (meaning I can run `docker` or they have `dpcpp` installed) have  A100.

Reply:
I get errors about invalid kernel arguments that I don't understand well since I'm already safeguarding work-group sizes with respect to local memory usage, and maximum kernel workgroup size. These don't appear on an A100.